prusti-tests/tests/cargo_verify/test_no_std/Prusti.toml

```diff
@@ -0,0 +1 @@
+be_rustc = true
```

Comment:
Can you explain this? Does this not just skip Prusti altogether?

Reply:
Yeah, but I don't think this test requires us to run Prusti on it at all, right @Pointerbender? I tried it out with removing the `#![no_std]` from `prusti-contracts` and it failed as expected, so assumed that it's all good.

Reply:
The reason I had to add this is that with the new changes (compiling `prusti-contracts` with `cargo-prusti`) one can no longer run Prusti on a crate with the dependency `prusti-contracts = { path = "path/to/prusti-contracts/prusti-contracts" }` since you'll get the following error:
```
error: [Prusti: verification error] statement might panic
   --> .../prusti-dev/prusti-contracts/prusti-contracts/src/lib.rs:395:5
    |
395 |     unimplemented!()
    |     ^^^^^^^^^^^^^^^^
    |
    = note: this error originates in the macro `unimplemented` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `prusti-contracts` due to previous error
```
Which is complaining about this fn:
```
#[trusted]
pub fn snap<T>(_x: &T) -> T {
    unimplemented!()
}
```
Instead Prusti will only work on crates with `prusti-contracts = { path = "path/to/prusti-contracts/prusti-contracts", features = ["prusti"] }`. Without the `prusti` feature the `#[trusted]` becomes transparent and Prusti complains about the `unimplemented!()`

Reply:
> Yeah, but I don't think this test requires us to run Prusti on it at all, right @Pointerbender? I tried it out with removing the `#![no_std]` from `prusti-contracts` and it failed as expected, so assumed that it's all good.

@JonasAlaif Yep, that's right! (Apologies for the late answer, I just got back from holiday :D )